Frameworks/UIKit/UIButton.mm

```diff
@@ -378,7 +378,7 @@ - (CGRect)imageRectForContentRect:(CGRect)contentRect {
 - (CGRect)titleRectForContentRect:(CGRect)contentRect {
     CGSize titleSize = [self.currentTitle sizeWithFont:self.font];
     CGSize totalSize = titleSize;
-    CGSize imageSize = [self.currentImage size];
+    CGSize imageSize = self.currentImage ? [self.currentImage size] : CGSizeZero;
```

Comment:
I'd prefer not to litter this code with comments, since this & similar patterns are common in here.  Yea CGSizeZero := CGSize{0, 0}